### DISCUSSION TITLE
Tab without support

### DISCUSSION THREAD

**Discussion by Shokishy** [kennetek/gridfinity-rebuilt-openscad#73] [Ideas]
Would it be possible to add an option for a tab without support ? 
For small bins, the printer can bridge over the bin and the 45° angle is not necessary.

Why ? To save time printing, material, and also to leave more space for stuff inside.

![image](https://user-images.githubusercontent.com/126019476/220446207-34aa0504-c80f-4f46-917c-92f1a222bdf1.png)

**Comment by kennetek** [comment]
Yeah, I think this would be a good feature. It may or may not be difficult to implement depending on how robust the cutters are. I will look into it. Creating an issue for this.

**Comment by Ruharc** [comment]
Could you just change the angle of that tab to 1 degree?  That would effectively make it not exist underneath the label tab.  You can adjust that value in the constants file.

**Reply by Shokishy** [reply]
If you change the tab angle to something low it just disappears.
